assets/app/views/project.html

```diff
@@ -20,21 +24,30 @@
             Get Started
           </a>
         </div>
+
+        <!-- Normal empty message (loading... or filtered) -->
         <div ng-if="!renderOptions.showGetStarted">
           <em>{{emptyMessage}}</em>
         </div>
+
       </div>
 
+
+
       <!-- TODO handle multiple services mapping to the same deploymentConfig/deployment/pod -->
-      <section ng-repeat="(serviceId, service) in services">
-        <div class="row">
-          <div class="col-md-12">
-            <div class="tile">
+      <section ng-repeat="(serviceId, service) in services" class="components components-group">
+
+        <div class="osc-object components-panel service" ng-init="numPorts = service.spec.ports.length" kind="Service" resource="service">
 
-              <!-- Routes -->
-              <h2 class="route osc-object" ng-if="displayRouteByService[serviceId]" ng-init="otherRoutes = (routesByService[serviceId] | hashSize) - 1" kind="Route" resource="displayRouteByService[serviceId]">
+          <div class="component-block">
+            <div class="component">
+              <div class="component-label">
+                Service <span ng-if="displayRouteByService[serviceId]">: {{serviceId}}</span>
+              </div>
+
+              <!-- Show the route if present -->
+              <h2 ng-if="displayRouteByService[serviceId]" ng-init="otherRoutes = (routesByService[serviceId] | hashSize) - 1">
```

Comment:
hrmmm... not sure what to do now that we collapse service and route into a single container, and the core part of the route is a clickable link